addons/hide-flyout/addon.json

```diff
@@ -53,6 +53,10 @@
           "id": "hover",
           "name": "Hover"
```

Comment:
Maybe this could have a better name now that "category hover" is also an option? What about `Block palette area hover` or similar?
These options can be confusing, so maybe we should add notices that better explain what each option does?

Reply:
I agree. I'm not so good with names, so I'll probably not have the best ideas.